components/siteHeader/index.js

```diff
@@ -0,0 +1,52 @@
+import { rem } from 'polished'
```

Comment:
we've been using lower-case for files (should be auto linted, but not sure why it's not working -- cc @julie-huang)

Reply:
Please change before landing @jasondippel. We can look into linting the file names in another PR.